src/generators/java/presets/JacksonPreset.ts

```diff
@@ -27,13 +27,19 @@ export const JAVA_JACKSON_PRESET: JavaPreset = {
       //Properties that are dictionaries with unwrapped options, cannot get the annotation because it cannot be accurately unwrapped by the jackson library.
       const isDictionary =
         property.property instanceof ConstrainedDictionaryModel;
+      const isAdditionalProperties =
+        property.propertyName === 'additionalProperties';
       const hasUnwrappedOptions =
         isDictionary &&
         (property.property as ConstrainedDictionaryModel).serializationType ===
           'unwrap';
 
       const blocks: string[] = [];
 
+      if (isAdditionalProperties) {
```

Comment:
You can use `hasUnwrappedOptions` instead.

```suggestion
      if (hasUnwrappedOptions) {
```

Reply:
moved the push into the existing if block for `hasUnwrappedOptions`